cybersyn/locale/sv-SE/base.cfg

```diff
@@ -0,0 +1,103 @@
+[mod-description]
+cybersyn=Använder cybernetiska kombinatorer för att skapa funktionsfyllda tågnätverk för logistik. Du kan koordinera de ekonomiska in- och utflödena för hela din megabas med bara den här modden.
+
+[mod-setting-name]
+cybersyn-enable-planner=Aktivera centralplanering
+cybersyn-ticks-per-second=Centralplaneringsuppdateringar per sekund
+cybersyn-update-rate=Centralplaneringuppdateringsfrekvens
+cybersyn-request-threshold=Standardbeställningströskel
+cybersyn-priority=Standardprioritet
+cybersyn-locked-slots=Standardvärde för låsta platser per lastvagn
+cybersyn-network-flag=Standardnätverksmask
+cybersyn-fuel-threshold=Bränsletröskel
+cybersyn-depot-bypass-enabled=Hoppa över depåer
+cybersyn-warmup-time=Stationsuppvärmningstid (sek)
+cybersyn-stuck-train-time=Timeout för tåg som fastnat (sek)
+cybersyn-allow-cargo-in-depot=Tillåt last i depåer
+cybersyn-invert-sign=Invertera kombinatorers utsignaler (förlegad)
+cybersyn-manager-enabled=Aktivera Cybersyn-GUI:t
+cybersyn-manager-update-rate=Uppdateringsfrekvens för hanteraren
+cybersyn-manager-result-limit=Max antal entiteter som visas på GUI-sidor.
+
+[mod-setting-description]
+cybersyn-enable-planner=Aktivera eller avaktivera centralplaneringsalgoritmen. När den är avstängd kommer ingen nya tåg att skickas iväg.
+cybersyn-ticks-per-second=Antal gånger per sekund som centralplaneraren ska uppdatera nätverksstatusen och planera leveranser. Värdet rundas upp till en divisor 60. Om satt till 0 stoppas alla avsändningar.
+cybersyn-update-rate=Antal stationer per tick som uppdateras eller får leveranser planerade åt gången. Större nummer låter centralplaneraren hålla sig uppdaterade gentemot nätverkets status, men kräver mer prestanda.
+cybersyn-request-threshold=Standardvärde för förfrågningströskeln när en specifik tröskelsignal inte ges till en station. När en station tar emot en negativ varusignal som är större än förfrågningströskeln, förutsatt att det finns en annan station med en positiv signal som är större än tröskeln, planeras en leverans av varan mellan de två stationerna.
```

Comment:
The use of _vara_ here and in other strings makes it sounds like there is some trading going on. Not sure what else to use though, _föremål_ or _last_ perhaps?

Reply:
"Last" could work maybe. I chose "vara" since the other terminology kinda fits with treating it like trade items (requesting, delivering, et.c).